templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  
+  <form action="{% url 'login' %}" method="POST">
```

Comment:
The 'login' URL is used here. Ensure that the URL pattern for login is correctly defined in your project's URL configuration to avoid any NoReverseMatch errors.